src/routes/index.js

```diff
@@ -15,6 +15,7 @@ router.get('/items/:id', getItem)
 router.post('/items', postItem)
 router.put('/items/:id', editItem)
 
-router.get('/checkTokenExp', checkToken, home)
+router.get('/chkToken', auth)
+router.get('/tokenTest', authMid, home)
```

Comment:
What are the purpose of these two apis?
`/chkToken` and `/tokenTest`.
Shouldn't we check token in the apis of POST '/items' and PUT '/items/:id' ?

Reply:
auth function will return the data which is encoded in the token. (/chkToken)
authMid function is middleware for us to check token in the APIs of POST '/items' and PUT '/items/:id'
/tokenTest is just a testing API which can be removed

Reply:
Ok. my questions is that what is the purpose of exporting the api `/chkToken` to clients.
I thought we are going to use the middleware code `auth` inside of the existing apis we need to check the authorization.

Reply:
I create "chkToken" because I need to check the token when the web page is reloaded.
The token in the local storage needs to be check is expired or not.
However, there is an issue so I cannot add this case in the front-end yet.

Reply:
Aha. Thanks for the clarification. I am going to merge this PR